README.md

```diff
@@ -80,10 +78,13 @@ Here is a sample config file:
       "pass": "s3",
       "fs": "s3",
       "params": {
+        "endpoint": "https://s3.amazonaws.com",
         "region": "eu-west-1",
         "bucket": "my-bucket",
         "access_key_id": "AKIA....",
         "secret_access_key": "IDxd...."
+        "disable_ssl": "false",
```

Comment:
lacking a comma here.

It's good, it just makes me wonder if I shouldn't also support any JSON type here (it would make the parsing code a bit more ugly but a cleaner config file).

Reply:
Since there is no prior strict data structure I wondered, it was kept intentionally free form. So that each driver implementation can handle their own params in their own ways.

Reply:
Let me if you need me to refactor this in some form.

Reply:
also, would it be possible to tag a release?